README.md

```diff
@@ -1,6 +1,6 @@
 # Cohort-middleware
 
-![version](https://img.shields.io/github/release/uc-cdis/cohort-middleware.svg) [![Apache license](http://img.shields.io/badge/license-Apache-blue.svg?style=flat)](LICENSE) [![Travis](https://travis-ci.org/uc-cdis/cohort-middleware.svg?branch=master)](https://travis-ci.org/uc-cdis/cohort-middleware) [![Coverage Status](https://coveralls.io/repos/github/uc-cdis/cohort-middleware/badge.svg?branch=master)](https://coveralls.io/github/uc-cdis/cohort-middleware?branch=master)
```

Comment:
I wanted to keep a failing badge for coverage so that we get triggered to fix it 😅

Reply:
Ok, that's fair. 😃 

Though, we're moving off Travis, so GitHub Action it is + Coveralls (which we still needs to do).